src/test/resources/inputs/builtins/nullif.sql

```diff
@@ -5,7 +5,12 @@ SELECT nullif(a, b) FROM T;
 SELECT nullif(b, a) FROM T;
 
 --#[builtins-nullif-02]
-SELECT nullif((nullif(a, b), c), d) FROM T;
```

Comment:
self-review: previous test was easy to misinterpret (and likely wrong) due to the extra parens. changed to be clearer and include the previous test for posterity